charts/brig/templates/configmap.yaml

```diff
@@ -55,7 +55,7 @@ data:
     emailSMS:
       email:
       {{- if .useSES }}
-        sesQueue: {{ .aws.sesQueue }}
+        sesQueue: {{ required "Missing value: brig.aws.sesQueue" .aws.sesQueue }}
         sesEndpoint: {{ .aws.sesEndpoint | quote }}
```

Comment:
We should also put this check for `sesEndpoint`, brig will fail with the same confusing error if `sesEndpoint` isn't defined.

Reply:
Hmm, then I guess it is ok. If someone sets it to `null`, it is on them :smile: